src/test/java/seedu/address/logic/commands/edit/EditInternshipCommandTest.java

```diff
@@ -1,2 +1,172 @@
+package seedu.address.logic.commands.edit;
 
+import static java.util.Objects.requireNonNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_ITEM;
+import static seedu.address.testutil.TypicalIndexes.INDEX_THIRD_ITEM;
+import static seedu.address.testutil.TypicalInternship.GOOGLE;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.commands.results.CommandResult;
+import seedu.address.model.item.Internship;
+import seedu.address.testutil.InternshipBuilder;
+import seedu.address.testutil.ModelStub;
+
+public class EditInternshipCommandTest {
```

Comment:
I think your PR is somehow intertwined with Hung's? Want to remove this?

Reply:
Did it in resolving merge conflicts.